modules/ticket_selector/TicketSelectorStandard.php

```diff
@@ -80,8 +80,8 @@ public function __construct(
      */
     protected function addTemplateArgs()
     {
-        $row                      = 1;
-        $ticket_row_html          = '';
+        $this->ticket_rows        = 0;
+        $all_ticket_rows_html     = '';
```

Comment:
$row started at 1 before because it was only used in the individual ticket row html and then incremented AFTER each row was generated. This approach doesn't work if we want to use that number afterwards for the total count of tickets because it always ends up one higher than the actual number of of tickets.